kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -512,6 +526,7 @@ define([
                 });
                 e.target.closest('li').remove();
                 updateDisabledFileValues();
+                updateDuplicateOutputValues();
```

Comment:
If a row gets deleted, we should set all duplicates again. That row might have had a problematic value, or would change which row was duplicated in the message.